productcomments.php

```diff
@@ -164,13 +164,34 @@ public function getCacheId($id_product = null)
     protected function _postProcess()
     {
         if (Tools::isSubmit('submitModerate')) {
-            Configuration::updateValue('PRODUCT_COMMENTS_MODERATE', (int) Tools::getValue('PRODUCT_COMMENTS_MODERATE'));
-            Configuration::updateValue('PRODUCT_COMMENTS_ALLOW_GUESTS', (int) Tools::getValue('PRODUCT_COMMENTS_ALLOW_GUESTS'));
-            Configuration::updateValue('PRODUCT_COMMENTS_USEFULNESS', (int) Tools::getValue('PRODUCT_COMMENTS_USEFULNESS'));
-            Configuration::updateValue('PRODUCT_COMMENTS_COMMENTS_PER_PAGE', (int) Tools::getValue('PRODUCT_COMMENTS_COMMENTS_PER_PAGE'));
-            Configuration::updateValue('PRODUCT_COMMENTS_ANONYMISATION', (int) Tools::getValue('PRODUCT_COMMENTS_ANONYMISATION'));
-            Configuration::updateValue('PRODUCT_COMMENTS_MINIMAL_TIME', (int) Tools::getValue('PRODUCT_COMMENTS_MINIMAL_TIME'));
-            $this->_html .= '<div class="conf confirm alert alert-success">' . $this->trans('Settings updated', [], 'Modules.Productcomments.Admin') . '</div>';
+            $errors = [];
+            $pcmt = Tools::getValue('PRODUCT_COMMENTS_MINIMAL_TIME');
```

Comment:
acronym are banned in clean code best practice could you replace it ? 
```suggestion
            $productCommentsMinimalTime = Tools::getValue('PRODUCT_COMMENTS_MINIMAL_TIME');
```

Reply:
In fact, this acronym is explained inline. If you change variable name at line 168, then you need change it at line 191 also.
And make the same change for $pcpp at line 176 and 192 are needed for consistency.